build.gradle

```diff
@@ -1612,9 +1612,12 @@ project(':test-common:test-common-api') {
     implementation project(':server')
     implementation project(':server-common')
     implementation project(':test-common')
-    implementation libs.junitJupiter
+    implementation libs.junitJupiterApi
+
     testImplementation libs.mockitoCore
+
     testRuntimeOnly libs.junitPlatformLanucher
```

Comment:
we don't need to declare `junitPlatformLanucher` again, right?

Reply:
That's right.